packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2742,4 +2742,29 @@ export default class Meeting extends StatelessWebexPlugin {
   unlockMeeting() {
     return MeetingUtil.unlockMeeting(this.inMeetingActions, this.meetingRequest, this.locusUrl);
   }
+
+  /**
+   * Sends DTMF tones to the current meeting
+   * @param {String} tones
```

Comment:
As a developer, I'm not sure what `tones` could be, can we create a shape/constants list of possible tones or a small documentation update to describe what possible tones can be used as this is marked a public method?

Reply:
Done, thanks for the feedback!